addons/account/models/account_move.py

```diff
@@ -116,6 +116,8 @@ def _sequence_year_range_monthly_regex(self):
         tracking=True,
         index='trigram',
     )
+    name_placeholder_form = fields.Char(compute='_compute_name_placeholder_form')
+    name_placeholder_list = fields.Char(default='/')
```

Comment:
Stored is a mistake, but the field is required because the static `placeholder="..."` is not usable in list views. Only way to have a placeholder in a list view is to use the dynamic placeholder combinaison `widget="char_with_placeholder_field" options="{'placeholder_field': 'field'}"` which is kinda overkill since it is used to display static data but yeah, it's required...